tests/time_stack_tests.py

```diff
@@ -0,0 +1,137 @@
+'''
+Copyright 2015 Planet Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import unittest
+import numpy
+import os
+
+from osgeo import gdal
+
+from radiometric_normalization import time_stack
+
+
+class Tests(unittest.TestCase):
+    def test_organise_images_to_bands(self):
+        # Seven images of five bands of 10 by 11 pixels
+        all_images = []
+        nodata = 2 ** 15 - 1
+        for i in range(7):
+            image = numpy.random.randint(
+                2 ** 15, size=(5, 10, 11)).astype('uint16')
+            all_images.append(image)
+
+        all_bands = time_stack._organise_images_to_bands(all_images, nodata)
+        no_bands = len(all_bands)
+        no_images = len(all_bands[0])
+        rows, cols = all_bands[0][0].shape
+
+        self.assertEqual(no_bands, 5)
+        self.assertEqual(no_images, 7)
+        self.assertEqual(rows, 10)
+        self.assertEqual(cols, 11)
+        for image in range(no_images):
+            for band in range(no_bands):
+                for row in range(rows):
+                    for col in range(cols):
```

Comment:
With this many levels of nesting, it's probably not a bad idea to use xrange instead of range